src/OrbitGl/App.cpp

```diff
@@ -545,16 +546,32 @@ void OrbitApp::ShowSourceCode(const orbit_client_protos::FunctionInfo& function)
           [this, module,
            function](const std::filesystem::path& local_file_path) -> ErrorMessageOr<void> {
             const auto elf_file = orbit_elf_utils::ElfFile::Create(local_file_path);
-            const auto line_info = elf_file.value()->GetLineInfo(function.address());
+            const auto first_line_or_error = elf_file.value()->GetLineInfo(function.address());
```

Comment:
I would suggest `first_line_info_or_error`. It matches `line_info` below better. Or just `line_info_or_error`.

Reply:
Done. Thanks.